routing_model/index.html

```diff
@@ -319,14 +320,34 @@ <h2>Deciding when to accept an incoming relocation</h2>
 
   <h2>Relocated node connection</h2>
   <div class=description>
-  <p>Manage node with NodeState=WaitingConnectionInfo and connection info request/response.<br/>
-    CandidateInfo is sent to NaeManager of the target interval<br/>
-    When we complete, we either stop responding to CandidateInfo RPCs, or we send NodeConnected RPC<br/>
+  <p>Manage node with NodeState=WaitingConnectionInfo and connection info request/response RPCs.<br/>
+    The candidate sends CandidateInfo to NaeManager of the target interval, so it will reach the managing section after split/merge.<br/>
```

Comment:
"The candidate sends CandidateInfo" -> "The candidate sends its CandidateInfo"

Reply:
"to NaeManager" -> "to the NaeManager"

Reply:
" so it will reach the managing section after split/merge." -> "to ensure it reaches a section even in the event of a split or a merge." ?